src/types/ProgressUpdates.d.ts

```diff
@@ -30,3 +30,13 @@ interface question {
 export interface formProps {
     questions: Array<question>;
 }
+
+export interface fetchedProgress {
+    id: string;
+    date: number;
```

Comment:
why date is in number

Reply:
As per the DTO decided by the backend, the API call returns a number
Also, the date constructor uses a number as a parameter.